browser/profiles/brave_profile_manager_unittest.cc

```diff
@@ -12,13 +12,15 @@
 #include "base/files/scoped_temp_dir.h"
 #include "base/strings/utf_string_conversions.h"
 #include "brave/browser/profiles/tor_unittest_profile_manager.h"
+#include "brave/browser/tor/tor_launcher_factory.h"
```

Comment:
we should probably keep all the tor stuff here inside `ENABLE_TOR` since we don't currently have an implementation for android

Reply:
the same also applies for the tor-specific test files in gn

Reply:
fixed in 03268dc482e62d0ff2c0573fb193528ca1508a28